### ISSUE TITLE
Add CI integration

### ISSUE THREAD

**Issue by lechup** [EnviroMonitor/EnviroMonitorFrontend#6]
Add travis or any other CI intergation.

**Comment by lechup** [comment]
@zen while I was trying to integrate Travis I was asked to request access to EnviroMonitor group - You should get an email about that.

**Comment by zen** [comment]
I don't see any request :(

**Comment by lechup** [comment]
@zen You've or someone else had clicked it already. I'm now able to continue integration. Thanks!

**Comment by lechup** [comment]
@zen Hm... Actually after integrating GitHub with Travis I'm getting:

```
This is not an active repository

You don't have sufficient rights to enable this repo on Travis. 
Please contact the admin to enable it or to receive admin rights yourself.
```

I think You should enable integration and then I would be able to continue...

**Comment by zen** [comment]
I did whatever what I was asked for. What else I can do?

**Comment by lechup** [comment]
Ohm. I wasn't clear enought. Please turn on TravisCI integration on TravisCI panel. I do not have enought rights.

Here is what I get from TravisCI:
![travisCI](https://cloud.githubusercontent.com/assets/957803/21228809/7e57c674-c2df-11e6-8137-61c5abaade33.jpg)

**Comment by zen** [comment]
Got it, I'm having problems adding EnviroMonitor to Travis, it just don't see those repose despite being authorized. Will try to figure that out

**Comment by zen** [comment]
Should be sorted out now. Please confirm.

**Comment by lechup** [comment]
@zen it's ok now. Thanks!

**Comment by lechup** [comment]
Closed by #13
